workspaces/ui-v2/src/contexts/MetadataLoader.tsx

```diff
@@ -24,7 +24,7 @@ export const MetadataLoader: FC = ({ children }) => {
   }
   if (result.error) {
     console.error(result.error);
-    return <>error</>;
+    return <>error loading app metadata</>;
```

Comment:
We'll want to start handling errors explicitly everywhere we expect them, as well as tighten our typescript / gql schemas so we aren't casting or using `any`. Giving a more specific error message here with better typings will help us more quickly identify where the errors are coming from (rather than the catch all)

Also, there's separate work to make the error states look nicer (i.e. not a single line of text) - but I think that'll come later